package.json

```diff
@@ -10,6 +10,7 @@
     "@wikia/ad-engine": "git+https://github.com/Wikia/ad-engine.git#v10.0.0",
     "core-decorators": "^0.20.0",
     "events": "^1.1.1",
+    "js-cookie": "^2.1.4",
```

Comment:
Is it safe? :trollface:

Reply:
Snyk says it's safe: [![Known Vulnerabilities](https://snyk.io/test/github/wikia/ad-products/badge.svg?targetFile=package.json)](https://snyk.io/test/github/wikia/ad-products?targetFile=package.json)

Reply:
I'm not sure, I moved this module from f2 repository, where this package [is used](https://github.com/Wikia/f2/blob/master/frontend/package.json#L32)